recipe/meta.yaml

```diff
@@ -86,6 +88,7 @@ outputs:
         - libcblas   # [not x86]
         - liblapack   # [not x86]
         - llvm-openmp    # [osx]
+        - libuv  # [win]
```

Comment:
do you think we should be linking to libuv on linux too?

Reply:
`libuv` is installed on Linux even though it's not explicitly in the recipe because CMake depends on it, see https://github.com/conda-forge/cmake-feedstock/blob/15be5198451d9a58e1ca2dcd9a5df7cd6fc4949c/recipe/meta.yaml#L52

Reply:
Therefor it should not be specific to windows right? This would help with the run-exports of uv if left in.